CMakeLists.txt

```diff
@@ -61,6 +61,14 @@ else(Trilinos_BUILD_SHARED_LIBS)
   set(Nalu_LIBRARY_TYPE STATIC)
 endif(Trilinos_BUILD_SHARED_LIBS)
 
+############################ BOOST #####################################
+find_package(Boost REQUIRED COMPONENTS filesystem)
+if (DEFINED Boost_LIBRARIES)
```

Comment:
`find_package` with `REQUIRED` will throw a fatal error, so you don't need to check it manually as you are doing :)

Reply:
Haha, you're right.  I wanted to be doubly-sure that it was linking to something that it found, but yes, the manual check is unnecessary.